.buildkite/scripts/pipelines/security_solution_quality_gate/create_periodic_test_docker_image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+if [ -z "${KIBANA_CURRENT+x}" ] || [ "$KIBANA_CURRENT" = "0" ]; then
+    echo "As we not testing against latest kibana image, this step is exiting with exit code 0"
+    exit 0
+fi
+
+
+.buildkite/scripts/bootstrap.sh
+
+source .buildkite/scripts/steps/artifacts/env.sh
+
+GIT_ABBREV_COMMIT=${BUILDKITE_COMMIT:0:12}
+KIBANA_IMAGE_TAG="git-$GIT_ABBREV_COMMIT"
```

Comment:
Can we rename the tag so it doesn't conflict with development promotion image names?  e.g. replacing `git` with `security-solution-quality-gate` or an abbreviation?

Reply:
Renamed to 'sec-sol-qg-${BUILDKITE_COMMIT:0:12}'